js/cli/example-configs/namespace/initializeNamespace.json

```diff
@@ -6,24 +6,12 @@
   "prettyName": "Trash with Frens",
   "desiredNamespaceArraySize": 2,
   "permissivenessSettings": {
-    "namespacePermissiveness": {
-      "namespace": true
-    },
-    "itemPermissiveness": {
-      "whitelist": true
-    },
-    "playerPermissiveness": {
-      "all": true
-    },
-    "matchPermissiveness": {
-      "whitelist": true
-    },
-    "missionPermissiveness": {
-      "blacklist": true
-    },
-    "cachePermissiveness": {
-      "all": true
-    }
+    "namespacePermissiveness": "ALL",
```

Comment:
The intended format of these CLI json configs is to match the format that comes down the wire using the IDL format when specifying enum values.

Reply:
Cool, mentioned above I like this idea, I was thinking I should wait until your PR is merged then make those changes